src/commonmark/en/content/user/configure-metadata.md

```diff
@@ -621,6 +621,18 @@ objects:
     element. Be as precise as possible and include complete information
     about how the data element is measured and what its purpose is.
 
+8. (Optional) In the **Field mask** field, you may type a template that's used to provide
+   hints for correct formating of the data element in Capture and Tracker Capture apps.
+   The following are special characters that can be used in the mask to match more than one character.
```

Comment:
If they can be used to match "more than one character", shouldn't the match be "digit(s)", "lower case letter(s)", etc?

Also, to be precise I think you probably want to way "one or more characters" rather than "more than one character", no?

Reply:
(I might also be useful with a couple of examples)

Reply:
I was actually quite unsure how to phrase it, as the special characters match exactly one character of the given type. Ie #### is exactly 4 digits. Examples will be added.

Reply:
OK. then instead of:
> The following are special characters that can be used in the mask to match more than one character.

I would probably try:
> The following are special characters that can be used in the mask. The special characters match exactly one character of the given type.

Reply:
(sometimes things like this are clearer when you break it down into smaller sentences)